spec/index.adoc

```diff
@@ -1674,6 +1674,96 @@ Any user can top-up any canister this way.
 
 This method is only available in local development instances.
 
+
+[#ic-bitcoin-canisters]
+== The IC bitcoin canisters
+
+Similar to the <<ic-management-canister,management canister>>, the Bitcoin functionality is also exposed through a facade, which we refer to as the "Bitcoin canisters".
+There are two Bitcoin canisters differing only in that one operates on the real Bitcoin blockchain whereas the other only works with the Bitcoin testnet.
+The Bitcoin canisters can be accessed using their addresses:
+
+- The canister address of the Bitcoin canister that uses the real Bitcoin blockchain is `TBD`.
```

Comment:
I'd like @bogwar and @akhi3030 to chime in here as well, but IMO we could choose a sub-range from the `01` type of principals. Currently we use everything with `nat64 | 0101` as a typical canister id, and I could imagine using `nat64(i) | 0201` for the Bitcoin canisters, with `i in {0,1}`.

Reply:
Could you not use the IC Management Canister to obtain the optional Bitcoin canister, with the behaviour depending on the subnet?

Reply:
I don't understand your proposal. What do you want to "obtain"? What exactly would the IC Management Canister return?

Reply:
> I don't understand your proposal. What do you want to "obtain"? What exactly would the IC Management Canister return?

Sorry, didn't see your question. 
I'm suggesting the ManagementCanister exposes an endpoint for getting the bitcoin service, perhaps as an option if there isn't one. Something like (in pseudo-candid).

```
service IC {
    get_bitCoin : () -> BitCoin opt 
}

service BitCoin {
     ....
}
```
So then you don't hard-code the canister ID but need to obtain it from the system.